tests/test_physical_channel_property.py

```diff
@@ -0,0 +1,110 @@
+"Contains a collection of pytest tests that validates the physical channel properties."
+import numpy
+import pytest
+
+from nidaqmx.constants import TerminalConfiguration
+
+
+def test__physical_channel__get_bool_property__returns_value(any_x_series_device):
```

Comment:
FYI, "given physical channel" does make sense, because that is an object that the test is given.